xcdat/regridder/regrid2.py

```diff
@@ -113,7 +114,7 @@ def _regrid(
     lon_mapping, lon_weights = _map_longitude(src_lon_bnds, dst_lon_bnds)
 
     # convert to pure numpy
-    input_data = input_data_var.astype(np.float32).data
+    input_data = input_data_var.astype(np.float32).values
```

Comment:
Make sure to use `.values` to convert Dask Arrays to `.ndarray`. Otherwise we get a `TypeError: take() got an unexpected keyword argument 'mode'` downstream with `np.take()`.